projects/aca-content/src/lib/components/dl-custom-components/name-column/name-column.component.ts

```diff
@@ -86,6 +95,10 @@ export class CustomNameColumnComponent extends NameColumnComponent implements On
         this.isFileWriteLocked = isLocked(this.node);
         this.cd.detectChanges();
       });
+
+    this.appExtensionService.getBadges(this.node).subscribe((badges) => {
```

Comment:
should probably have "takeUntil(destroy)"

Reply:
Added